### PR TITLE
Assorted Refinements

### PR DESCRIPTION
**Pull request by MrGadget1024**
image

### CODE REVIEW

File: ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.3.6f1
+m_EditorVersion: 2018.4.15f1
```
Comment (MrGadget1024) [inline ProjectSettings/ProjectVersion.txt]:
```suggestion
m_EditorVersion: 2018.3.6f1
```

File: Assets/Mirror/Discovery/NetworkDiscovery.cs [hunk 2]
```diff
@@ -1,31 +1,30 @@
...
     public class NetworkDiscovery : NetworkDiscoveryBase<ServerRequest, ServerResponse>
     {
         #region Server
-        public long ServerId { get; private set; }
 
-        public ServerFoundUnityEvent ServerFound;
+        public long ServerId { get; private set; }
 
-        [Tooltip("Transport exposed for discovery")]
+        [Tooltip("Transport used for discovery")]
```
Comment (paulpach) [inline Assets/Mirror/Discovery/NetworkDiscovery.cs]: This is a bit confusing,   we always use UDP broadcast for discovery,  not the transport.  We just make that transport discoverable, we don't actually use it.

File: Assets/Mirror/Discovery/NetworkDiscovery.cs [hunk 3]
```diff
@@ -44,6 +43,8 @@ public void Start()
         /// Override if you wish to provide more information to the clients
         /// such as the name of the host player
         /// </remarks>
+        /// <param name="_"></param>
```
Comment (paulpach) [inline Assets/Mirror/Discovery/NetworkDiscovery.cs]:
```suggestion
        /// <param name="_">Request coming from the client</param>
```

File: Assets/Mirror/Discovery/NetworkDiscovery.cs [hunk 4]
```diff
@@ -44,6 +43,8 @@ public void Start()
...
+        /// <param name="endpoint"></param>
```
Comment (paulpach) [inline Assets/Mirror/Discovery/NetworkDiscovery.cs]:
```suggestion
        /// <param name="endpoint">The UDP endpoint of the client that sent the request</param>
```

**Pull request merged**